desktop/src/main/nl/tudelft/ti2806/riverrush/screen/LoadingScreen.java

```diff
@@ -56,17 +55,17 @@ public void show() {
         this.stage.addActor(image);
 
         //Load all images/textures that we will use
-        this.assets.load(getFileName("boat.jpg"), Texture.class);
-        this.assets.load(getFileName("shipv2.png"), Texture.class);
-        this.assets.load(getFileName("ship.png"), Texture.class);
-        this.assets.load(getFileName("raccoon.png"), Texture.class);
-        this.assets.load(getFileName("pirateship.png"), Texture.class);
-        this.assets.load(getFileName("left.jpg"), Texture.class);
-        this.assets.load(getFileName("grass.jpg"), Texture.class);
-        this.assets.load(getFileName("river.jpg"), Texture.class);
-        this.assets.load(getFileName("cannonball.png"), Texture.class);
-        this.assets.load(getFileName("win.png"), Texture.class);
-        this.assets.load(getFileName("lose.png"), Texture.class);
+        this.assets.load(Assets.boat, Texture.class);
+        this.assets.load(Assets.shipv2, Texture.class);
+        this.assets.load(Assets.ship, Texture.class);
+        this.assets.load(Assets.raccoon, Texture.class);
+        this.assets.load(Assets.pirateship, Texture.class);
+        this.assets.load(Assets.left, Texture.class);
+        this.assets.load(Assets.grass, Texture.class);
+        this.assets.load(Assets.river, Texture.class);
+        this.assets.load(Assets.cannonball, Texture.class);
+        this.assets.load(Assets.win, Texture.class);
+        this.assets.load(Assets.lose, Texture.class);
```

Comment:
You could move these calls into the Assets class since loading all assets is not the responsibility of a Screen class